.travis.yml

```diff
@@ -5,8 +5,8 @@ matrix:
     language: swift
     osx_image: xcode10.1
     before_script: cd iOS/FVM
-    script: xcodebuild -project FVM.xcodeproj -scheme FVM -sdk iphonesimulator -destination 'platform=iOS Simulator,OS=12.1,name=iPhone X' test -configuration Debug ONLY_ACTIVE_ARCH=YES -enableCodeCoverage YES | xcpretty
```

Comment:
What was **xcpretty** doing?

Reply:
**xcpretty** was formatting Travis's logs. Unfortunately the result from entire command always returned 0 so I've decided to remove it.